src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Media/ImageBrushTests/ImageBrush_Shape_Fill_StretchAndAlignment.cs

```diff
@@ -22,71 +22,78 @@ public partial class ImageBrush_Shape_Fill_StretchAndAlignment : SampleControlUI
 
 		[Test]
 		[AutoRetry]
-		[ActivePlatforms(Platform.Browser)]
-		//Only include Browser as a platform until following issue is fixed: https://github.com/unoplatform/uno/issues/5271
 		public void When_StretchAndAlignment()
 		{
-			Run("UITests.Windows_UI_Xaml_Media.ImageBrushTests.ImageBrushShapeStretchesAlignments");
-			_app.WaitForElement("MyRectangle");
-
+			try
+			{
+				Run("UITests.Windows_UI_Xaml_Media.ImageBrushTests.ImageBrushShapeStretchesAlignments");
+				_app.SetOrientationPortrait();
```

Comment:
why?

Reply:
Otherwise some of the UI elements are off the screen for some of the test cases so the ui test wouldn't find them and I was too lazy to change the test XAML around

Reply:
This would mean you need to add a `.All()` to your queries.